src/main/java/com/softserve/academy/spaced/repetition/repository/DeckRepository.java

```diff
@@ -38,4 +39,8 @@
     Deck getDeckByItsIdAndOwnerOfDeck(@Param("deck_id") Long deck_id, @Param("user_id") Long user_id);
 
     List<Deck> findAllByDeckOwner_IdEquals(Long user_id);
+
+    Page<Deck> findAllByCategoryEquals(Category category, Pageable pageable);
+
+    Page<Deck> findAllByOrderByRatingDesc(Pageable pageable);
```

Comment:
Can it be renamed to findAllOrderByRatingDesc ?

Reply:
this method "Page <Deck> findAllByOrderByRatingDesc (Pageable pageable);" is not used I will correct it